php/php.editor/src/org/netbeans/modules/php/editor/csl/FoldingScanner.java

```diff
@@ -131,6 +131,7 @@
     );
 
     private static final String LAST_CORRECT_FOLDING_PROPERTY = "LAST_CORRECT_FOLDING_PROPERY"; //NOI18N
+    private static final boolean FOLD_PHPTAG = !Boolean.getBoolean("nb.php.editor.do.not.fold.phptag"); // NOI18N NETBEANS-5480
```

Comment:
Nitpick: I would perhaps prefer `nb.php.editor.doNotFoldPhptag` :) But no need to change it, of course!

Reply:
Will fix it soon. Thanks!

Reply:
Fixed.